packages/shared/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@rehearsal/shared",
```

Comment:
Could you rename package to `@rehearsal/plugins` + move files under `helpers/` to `@rehearsal/utils`?

Reply:
Thanks for suggestion. Having a plugins package we can create a suite of tools around testing too.